doc/scripts/skip_headers

```diff
@@ -0,0 +1,8 @@
+SolveLinear2.h
+Unrolling.h
+MinCostMaxFlow.h
+DivideAndConquerDP.h
+ModInverse.h
+ModSqrt.h
+Hashing-codeforces.h
+NumberTheoreticTransform.h
```

Comment:
The somewhat unelegant part of this is that it lives in a different part of the pipeline than the header parsers.

So the easiest way to do it would be to grep for a specific string - which might not be too bad.